config/vufind/Folio.ini

```diff
@@ -142,3 +142,10 @@ maxNumberItems = 5
 ; dueDates will be shown in Universal Time Format, e.g. 06 Jul 2022 14:59, set showTime
 ; to false to show 06 Jul 2022 only
 showTime = false
+
+[Holds]
```

Comment:
We should probably add comments here to explain what these settings mean... e.g., before `[Holds]`:

`; This section associates status text found in FOLIO with specific VuFind statuses`

...and then maybe a comment above each of the settings as well ("FOLIO statuses indicating available-for-pickup items" / "FOLIO statuses indicating in-transit items").

Reply:
@cwolfebsco, thanks for adding the comments, but I just realized something -- there is already a `[Holds]` section in Folio.ini, and having two is going to cause problems. You should move your new settings into the existing section. Maybe you can adjust your "This section" comment to instead say "The following settings" since it no longer describes the entire section.

Reply:
Oh that is bad and embarrassing, I'll fix this ASAP